src/convert-package.ts

```diff
@@ -60,16 +60,6 @@ async function setupOutDir(outDir: string, clean = false) {
   }
 }
 
-/**
- * Create the default conversion settings, adding any "main" files from the
- * current package's bower.json maifest to the "includes" set.
- */
-function getConversionSettings(
-    analysis: Analysis, options: PackageConversionSettings) {
-  const conversionSettings = createDefaultConversionSettings(analysis, options);
-  return conversionSettings;
-}
-
```

Comment:
I was going to update the comment, then I realized that this function does nothing so I just removed it...

Reply:
Ha! nice